mail-mta/protonmail-bridge-bin/protonmail-bridge-bin-1.0.4.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit rpm
+
+DESCRIPTION="Serves ProtonMail to IMAP/SMTP clients"
+HOMEPAGE="https://protonmail.com/bridge/"
+SRC_URI="https://protonmail.com/download/${P/-bin/}-1.x86_64.rpm"
+
+RESTRICT="bindist mirror"
+
+LICENSE="MIT protonmail-bridge-EULA"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
+RDEPEND="
+	app-crypt/libsecret
+	dev-libs/glib:2
+	media-sound/pulseaudio
+	virtual/opengl
+"
+
+S="${WORKDIR}"
+
+QA_PREBUILT="*"
+
+src_install() {
+	# Using doins -r would strip executable bits from all binaries
+	cp -pPR "${S}"/usr "${D}"/ || die "Failed to copy files"
+
+	dosym "Desktop-Bridge" "/usr/bin/${PN}" || die
```

Comment:
Are you covinced it should be named `protonmail-bridge-bin` and not just `protonmail-bridge` here?

Reply:
Actually, the problem of -bin packages is a bit different. There's firefox-bin because it's co-installable with source-built firefox (and that really makes sense because upstream doesn't support Firefox built from sources [or let it be called Firefox]).

AFAIU you plan to eventually replace this with source package; in this case, naming it -bin seems unnecessary and would only cause additional effort in the future.

Reply:
The -bin suffix has been added to some exclusively close source packages in the past (e.g. spideroak-bin before the company re-branded the product name to SpiderOakONE), so it seemed to be convention (though perhaps accidental).

Above, @csmk asked me to change the package to -bin, so I changed the name. If you prefer the package not named -bin, I'll let you two hash it out since I don't commit access.

If instead you are suggesting to name the package -bin but name the symlink w/o -bin, let me know, and I'll make that change (though how that avoids "additional effort in the future" isn't clear to me).

Reply:
When it's released as open-source, you'd either have to rename it (which is extra effort, and burns -bin variant forever) or create a second package (which means users of -bin won't migrate to open source vesion).

It's your choice.